File: src/IsbInterop/EDocuments/Wrappers/EDocumentInfo.vb
Imports IsbInterop.Base.Wrappers

Namespace EDocuments.Wrappers

    ''' <summary>
    ''' Обертка над IEDocumentInfo.
    ''' </summary>
    Friend Class EDocumentInfo
        Inherits ObjectInfo
        Implements IEDocumentInfo

        ''' <summary>
        ''' Конструктор.
        ''' </summary>
        ''' <param name="documentInfo">Объект с информацией о документе DIRECTUM.</param>
        ''' <param name="scope">Область видимости.</param>
        Public Sub New(documentInfo As Object, scope As IScope)
            MyBase.New(documentInfo, scope)
        End Sub
    End Class
End Namespace

File: src/IsbInterop.CSharp/EDocuments/Wrappers/EDocumentInfo.cs
using IsbInterop.Base.Wrappers;

namespace IsbInterop.EDocuments.Wrappers
{
  /// <summary>
  /// Обертка над IEDocumentInfo.
  /// </summary>
  internal class EDocumentInfo : ObjectInfo, IEDocumentInfo
  {
    /// <summary>
    /// Конструктор.
    /// </summary>
    /// <param name="rcwIEDocumentInfo">COM-объект IEDocumentInfo.</param>
    /// <param name="scope">Область видимости.</param>
    public EDocumentInfo(object rcwIEDocumentInfo, IScope scope) : base(rcwIEDocumentInfo, scope) { }
  }
}
